static/webpack.config.js

```diff
@@ -21,7 +21,7 @@ const FixStyleOnlyEntriesPlugin = require("webpack-fix-style-only-entries");
 const config = {
   entry: {
     choropleth: [
-      __dirname + "/js/tools/choropleth/choropleth.tsx",
+      __dirname + "/js/tools/choropleth/choropleth_template.tsx",
```

Comment:
This is already changed in `master`.

Reply:
I think this should avoid a merge conflict by adding it (hopefully)

Reply:
have you rebased with master?